nodes/03-Lev-002.xml

```diff
@@ -65,100 +66,100 @@
                       </Node>
                     </Node>
                   </Node>
                   <Node Cat="V" Rule="Vp2V" Head="0" nodeId="0300200100810012">
                     <Node Cat="vp" Rule="V2VP" Head="0" nodeId="0300200100810011">
-                      <Node n="030020010081" Cat="verb" morphId="030020010081" Unicode="יִהְיֶ֣ה" nodeId="0300200100810010">
-                        <m word="LEV 2:1!8" n="030020010081" lang="H" after=" " lemma="1961" morph="Vqi3ms" pos="verb" stem="qal" type="yiqtol" person="third" gender="masculine" number="singular">יִהְיֶ֣ה</m>
+                      <Node n="030020010081" Cat="verb" morphId="030020010081" Unicode="יִהְיֶ֣ה" nodeId="0300200100810010" StrongNumberX="1961" Frame="{A0:030020010091;}" Greek="ἔσται" GreekStrong="2071">
+                        <m word="LEV 2:1!8" n="030020010081" lang="H" after=" " lemma="1961" morph="Vqi3ms" pos="verb" stem="qal" type="yiqtol" person="third" gender="masculine" number="singular" english="be" mandarin="是">יִהְיֶ֣ה</m>
                       </Node>
                     </Node>
                   </Node>
                   <Node Cat="S" Rule="Np2S" Head="0" nodeId="0300200100910021">
                     <Node Cat="np" Rule="NPofNP" Head="0" nodeId="0300200100910020">
                       <Node Cat="np" Rule="N2NP" Head="0" nodeId="0300200100910011">
-                        <Node n="030020010091" Cat="noun" morphId="030020010091" Unicode="קָרְבָּנ" nodeId="0300200100910010">
-                          <m word="LEV 2:1!9" n="030020010091" morph="Ncmsc" lang="H" lemma="7133 a" pos="noun" type="common" gender="masculine" number="singular" state="construct">קָרְבָּנ֑</m>
+                        <Node n="030020010091" Cat="noun" morphId="030020010091" Unicode="קָרְבָּנ" nodeId="0300200100910010" StrongNumberX="7133" SenseNumber="1">
+                          <m word="LEV 2:1!9" n="030020010091" morph="Ncmsc" lang="H" lemma="7133 a" pos="noun" type="common" gender="masculine" number="singular" state="construct" english="offering" mandarin="供物">קָרְבָּנ֑</m>
                         </Node>
                       </Node>
                       <Node Cat="np" Rule="Pron2NP" Head="0" nodeId="0300200100920011">
-                        <Node n="030020010092" Cat="pron" morphId="030020010092" Unicode="֑וֹ" nodeId="0300200100920010">
-                          <m word="LEV 2:1!9" n="030020010092" morph="Sp3ms" lang="H" after=" " pos="suffix" type="pronominal" person="third" gender="masculine" number="singular">וֹ</m>
+                        <Node n="030020010092" Cat="pron" morphId="030020010092" Unicode="֑וֹ" nodeId="0300200100920010" StrongNumberX="2050c" Ref="{030020010012}" Greek="αὐτοῦ" GreekStrong="846">
```

Comment:
What is `@Ref` for? I'm guessing it's a reference to one of the resources, but I'm not sure which.